### DISCUSSION TITLE
`postcss-minify` may not be the best name

### DISCUSSION THREAD

**Discussion by ehoogeveen-medweb** [csstools/postcss-plugins#1115] [General]
I wanted to try out the new [@csstools/postcss-minify](https://www.npmjs.com/package/@csstools/postcss-minify), but I didn't read carefully and ended up installing [postcss-minify](https://www.npmjs.com/package/postcss-minify) instead. It also worked okay as I guess the tools have more or less the same goal, but I'm sure there are some differences that might eventually lead to confusing bug reports :)

I didn't even think to check, as [postcss-preset-env](https://www.npmjs.com/package/postcss-preset-env) is not a namespaced package.

**Comment by romainmenke** [comment]
Hi @ehoogeveen-medweb,

Thank you for reaching out about this.

I wonder if it would be sufficient to improve our documentation?  
The human readable name we use in docs never includes the namespace, maybe changing this would be better?

https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-minify#readme

old :

<img width="614" alt="Screenshot 2023-09-11 at 12 31 53" src="https://github.com/csstools/postcss-plugins/assets/11521496/6a879266-8e71-4f66-bb58-d31ee3a0b6ea">

possible improvement :

<img width="530" alt="Screenshot 2023-09-11 at 12 45 07" src="https://github.com/csstools/postcss-plugins/assets/11521496/8ddfe116-d67a-466f-a2dc-483b6833bd80">


------

I am hoping that the `@csstools` scope is sufficient and that we can keep using the best name for plugins, even when an unscoped plugin might exist under that same name.

> I didn't even think to check, as [postcss-preset-env](https://www.npmjs.com/package/postcss-preset-env) is not a namespaced package.

Correct, all the older plugins are unscoped and everything from the past ±2 years has been scoped.  
Ideally we one day have only scoped packages under `@csstools/` but I don't know (yet) how we can make this minimally disruptive. Maybe dual publishing for a few years?

Having only scoped packages would make it clearer who the author/maintainer is without people needing to investigate further.

**Comment by ehoogeveen-medweb** [comment]
1. I think having installation instructions that can easily be copy/pasted is always helpful, since you can either use it verbatim if you're using `npm`, or tweak it if you're using a different package manager. Even if you only use it as a reference but type it by hand, you're probably going to notice the package scope :)
2. I don't know how useful adding `CSSTools` to the link text would be, since it could just as easily be interpreted as branding (rather than hinting that a non-CSSTools variant exists). I don't think it hurts anything either though.
3. Having scoped packages for everything in the CSSTools monorepo would be helpful, I think, since if you're already using e.g. `@csstools/postcss-preset-env`, you're more likely to remember the scope for `@csstools/postcss-minify`. I think it also just looks better (and might help with sorting imports). It won't help existing users unless they can be migrated (or urged to migrate) though.

I don't think there's a perfect solution unfortunately. By the way, `postcss-minify` has version `1.1.0` so if you can come up with a reason to release a new major version of `@csstools/postcss-minify` so that its version number is visibly higher, that might also lessen the chance of confusion ;)

**Comment by romainmenke** [comment]
I've updated the documentation with only the install instructions a bit higher up in the readme.

I was suddenly not sure if adding `CSSTools` before the human readable name was the right direction.
I agree with you that it can be interpreted as branding and that might be confusing because not all packages we maintain have the scope and would get this prefix.

Thank you again for this feedback 🙇

**Comment by ehoogeveen-medweb** [comment]
Thanks!
